reference/configuration/framework.rst

```diff
@@ -524,6 +551,16 @@ For more details, see :doc:`/book/translation`.
 validation
 ~~~~~~~~~~
 
+.. _validation-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``false``, the option will be automatically
+enabled when :ref:`form support is enabled <form-enabled>`
```

Comment:
If this option behaves the same as `csrf_protection` option, we could make explanations consistent:

```
**type**: ``boolean`` **default**: ``true`` if form support is enabled, ``false`` otherwise.

**type**: ``boolean`` **default**: ``false``, the option will be automatically enabled when
form support is enabled
```